Octokit.Reactive/Clients/IObservablePullRequestsClient.cs

```diff
@@ -23,6 +23,11 @@ public interface IObservablePullRequestsClient
         IObservablePullRequestReviewCommentsClient ReviewComment { get; }
 
         /// <summary>
+        /// Client for managing review requests.
+        /// </summary>
+        IObservablePullRequestReviewRequestsClient ReviewRequests { get; }
```

Comment:
Our convention is to use plural for the client name (which you've done `PullRequestReviewRequestsClient`) but to use singular for the sub-client property accessors.  So in this case rather than `ReviewRequests` it should just be `ReviewRequest`